backend/src/club/club.gateway.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  SubscribeMessage, WebSocketGateway,
+  ConnectedSocket, WsResponse,
+  OnGatewayConnection, OnGatewayDisconnect,
+} from '@nestjs/websockets';
+import { Socket } from 'socket.io';
+import { Logger, UseGuards } from '@nestjs/common';
+import { RolesGuard } from '../roles/roles.guard';
+import { AllowedRoles } from '../roles/roles.decorator';
+import { Roles } from '../roles/roles.enum';
+import * as gatewayEvents from './gateway-events.json';
+import { WsJwtGuard } from '../authentication/ws.jwt.guard';
+
+@WebSocketGateway({ transports: ['polling'], origins: '*:*' })
```

Comment:
`transports: ['polling']` forces to use only long polling which is not what we want right? Transports can be omitted if we want to use both polling and sockets, see https://socket.io/docs/client-api/#With-websocket-transport-only and https://socket.io/docs/server-api/